components/dictaphone/Dictaphone.js

```diff
@@ -6,9 +6,12 @@ import {
   Box,
   Box1,
   ContainerMain,
+  Mic,
   Navbar1,
   ScreenBox,
+  Microphone,
   ScreenBox1,
+  Boutons,
```

Comment:
Boutons? This is a single "Button"

Reply:
it is a styled "button". i should remove the "s" at he end